crates/hdk/src/entry.rs

```diff
@@ -70,13 +72,19 @@ where
     create(CreateInput::try_from(input)?)
 }
 
-/// Alias to delete
+/// Delete an app entry. Also see [`delete`].
 ///
-/// Takes the [ `HeaderHash` ] of the element to delete.
+/// This function accepts the [`HeaderHash`] of the element to delete and optionally an argument to
+/// specify the [`ChainTopOrdering`]. Refer to [`DeleteInput`] for details.
 ///
 /// ```ignore
 /// delete_entry(entry_hash(foo_entry)?)?;
 /// ```
+///
+/// with a specific [`ChainTopOrdering`]:
+/// ```ignore
+/// delete_entry(DeleteInput::new(entry_hash(foo_entry)?, ChainTopOrdering::Relaxed)?;
```

Comment:
Despite the name, `delete_entry` only takes HeaderHashes, not EntryHashes. Other than that, this looks good and is correct.

Reply:
I think it's clear now.